lib/airbrake/sneakers.rb

```diff
@@ -10,12 +10,15 @@ class ErrorReporter
       # @see https://github.com/airbrake/airbrake/issues/850
       IGNORED_KEYS = %i[delivery_tag consumer channel].freeze
 
-      def call(exception, worker = nil, **context)
+      # rubocop:disable Style/OptionalArguments
+      def call(exception, worker = nil, context)
+        # Later versions add a middle argument.
```

Comment:
Could you elaborate on this? I'm not a Sneakers user myself and this integration was contributed by someone else.
I can see that Sneakers actually always passes 2nd argument:
https://github.com/jondot/sneakers/blob/7c059d1490e4dde43f98b801c38e4e0df7ac458c/lib/sneakers/error_reporter.rb#L18

Perhaps, we should just delete `= nil` so that the signature is:

`def call(exception, worker, context)`

Reply:
That won't work with some versions of sneakers. I'm not a sneakers user myself but I tracked down the code that calls this handler and reviewed its commit history. At one point they added a second argument, which they always pass in versions since then. For versions before then they only pass two arguments, which for this handler correspond to the first and last argument.

Reply:
Thanks, that makes sense.